cinema/urls.py

```diff
@@ -1 +1,28 @@
-# write urls here
+from django.urls import path, include
+from rest_framework.routers import DefaultRouter
+
+from cinema.views import (
+    GenreViewSet,
+    ActorViewSet,
+    MovieViewSet,
+    CinemaHallViewSet,
+    MovieSessionViewSet,
+)
+
+
+router = DefaultRouter()
+router.register("genres", GenreViewSet, basename="genres")
+router.register("actors", ActorViewSet, basename="actors")
+router.register("cinema_halls", CinemaHallViewSet, basename="cinema_halls")
+router.register("movies", MovieViewSet, basename="movies")
+router.register(
+    "movie_sessions",
+    MovieSessionViewSet,
+    basename="movie_sessions"
+)
```

Comment:
Keep router registrations consistent - use single-line format for all, or multi-line format for all.
Currently, `movie_sessions` differs from others.